lib/pubid/iso/parser.rb

```diff
@@ -57,7 +57,7 @@ class Parser < Parslet::Parser
     rule(:organization) do
       str("IEC") | str("IEEE") | str("CIW") | str("SAE") |
         str("CIE") | str("ASME") | str("ASTM") | str("OECD") | str("ISO") |
-        str("IWA") | str("HL7")
+        str("IWA") | str("HL7") | str("CEI")
```

Comment:
I don't think we should treat CEI has a different publisher. CEI is the french name of IEC.

Reply:
OK just saw the line https://github.com/metanorma/pubid-iso/pull/22/files#diff-1f7ba9cf7d0ae14071d13fb5a8b9f7a4425ab953b8f3874456850b92f463b245R42 it should be fine.